domaintools/api.py

```diff
@@ -354,17 +354,52 @@ def iris_investigate(self, domains=None, data_updated_after=None, expiration_dat
 
     def iris_detect_monitors(self, include_counts=False, datetime_counts_since=None, sort=None, order="desc", offset=0,
                              limit=None, **kwargs):
-        """Returns back a list of monitors based on the provided filters.
+        """Returns back a list of monitors in Iris Detect based on the provided filters.
         """
 
         if include_counts:
             if not datetime_counts_since:
                 raise ValueError('Need to define datetime_counts_since when include_counts is True')
             if isinstance(datetime_counts_since, datetime):
                 datetime_counts_since = str(datetime_counts_since.astimezone())
+            elif isinstance(datetime_counts_since, str):
+                kwargs["datetime_counts_since"] = datetime_counts_since
             kwargs["include_counts"] = "true"
             kwargs["datetime_counts_since"] = datetime_counts_since
         if sort:
             kwargs["sort[]"] = sort
         return self._results('iris-detect-monitors', '/v1/iris-detect/monitors/', order=order, offset=offset,
                              limit=limit, items_path=('monitors',), response_path=(), **kwargs)
+
+    def iris_detect_new_domains(self, monitor_id=None, escalation_types=None, tlds=None, risk_score_ranges=None,
+                                mx_exists=None, discovered_since=None, changed_since=None, search=None, sort=None,
+                                order=None, include_domain_data=False, offset=0, limit=None, preview=None, **kwargs):
+        """Returns back a list of new domains in Iris Detect based on the provided filters.
+        """
+        if discovered_since and changed_since:
+            raise ValueError('Please only use one of discovered_since or changed_since in your query.')
```

Comment:
From talking with Grant no reason both of these params should be used at the same time.